README.md

```diff
@@ -1,7 +1,7 @@
 - 👋 Hi, I’m @damnYouCode
 - 👀 I’m interested in learning git, SQL, and Python
-- 🌱 I’m currently learning SQL, Python and git,
-- 💞️ I’m looking to collaborate on ... who knows, I need to learn stuff first.
+- 🌱 I’m currently learning SQL, Python and git
+- 💞️ I’m looking to collaborate on ... on my new branch.
```

Comment:
Looks good to me 👍🏼

Reply:
```
$ git init
Initialized empty Git repository in /Users/skills/Projects/recipe-repository/.git/
```

Reply:
Here's what I wants:
- [ ] Turn on Github Pages
- [ ] Outline my portfolio 
- [ ] Introduce myself to the world!